### DISCUSSION TITLE
Character replacement :

### DISCUSSION THREAD

**Discussion by NicolasUrana** [DraqueT/PolyGlot#1252] [Ideas]
I propose that it is possible to enter more than one character in the character box.

For example that "a2" be replaced by "ɐ".

**Comment by rolfmeles** [comment]
This would be great. 
Some platforms use this kind of character replacements for Esperanto, as some people can't type ĉ or ŭ with their keiboards. These platforms then support a replacement of cx→ĉ and ux→ŭ.
Just to give an example that this feature would be really useful.
